packages/fast-usdc/test/exos/status-manager.test.ts

```diff
@@ -19,12 +41,32 @@ test('advancing creates new entry with ADVANCING status', t => {
 
   t.is(entries[0]?.status, PendingTxStatus.Advancing);
 });
-test.todo('ADVANCING transactions are published to vstorage');
 
-test('observe creates new entry with OBSERVED status', t => {
+test('ADVANCED transactions are published to vstorage', async t => {
   const zone = provideDurableZone('status-test');
-  const statusManager = prepareStatusManager(zone.subZone('status-manager'));
+  const statusManager = prepareStatusManager(
+    zone.subZone('status-manager'),
+    t.context.makeStatusNode,
+  );
 
+  const evidence = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
+  statusManager.advance(evidence);
+  // Flush event loop since status manager doesn't await vstorage write
+  await new Promise(res => setTimeout(res, 1));
```

Comment:
```suggestion
  // Flush event loop since status manager doesn't await vstorage write
  await eventLoopIteration();
```

you could even drop the comment, since `eventLoopIteration` is so often used to wait for vstorage writes to settle.

In fact, maybe we should alias it to `vstorageWritesQueisce()` or something.

Reply:
Done